index.bs

```diff
@@ -368,6 +370,160 @@ dictionary SanitizerConfig {
+specific element. For {{SanitizerConfig|Sanitizer configurations}}, it is assumed that
+configuration authors would prefer either all-global or all-local attribute declarations.
+If they do decide to mix them, the following rules apply. Note in particular that ambiguous
+configurations, where a particular attribute would be allowed by one list and forbidden by another,
+are generally invalid.
+
+<table class=data>
+<tr>
+ <td></td>
+ <th>global {{SanitizerConfig/attributes}}</th>
+ <th>global {{SanitizerConfig/removeAttributes}}</th>
+</tr>
+<tr>
+ <th>local {{SanitizerElementNamespaceWithAttributes/attributes}}</th>
+ <td class=long>An attribute is allowed if it matches either list. No duplicates are allowed.</td>
+ <td class=long>An attribute is allowed if it's in the local allow list.
+   No duplicate entries between global remove and local allow lists are allowed.
+   Note that the global remove list has no function for this particular element, but may well
+   apply to other elements that do not have a local allow list.</td>
+</tr>
+<tr>
+ <th>local {{SanitizerElementNamespaceWithAttributes/removeAttributes}}</th>
+ <td class=long>An attribute is allowed if it's in the global allow-list, but not in the local
+   remove-list. Local remove must be a subset of the global allow lists. </td>
+ <td class=long>An attribute is allowed if it is in neither list.
+   No duplicate entries between global remove and local remove lists are allowed.</td>
+</tr>
+</table>
+
+The {{SanitizerConfig/dataAttributes}} setting allows [=custom data attributes=]. The rules
+above easily extends to [=custom data attributes=] if one considers
+{{SanitizerConfig/dataAttributes}} to be an allow-list:
+
+<table class=data>
+<tr>
+ <td></td>
+ <th>global {{SanitizerConfig/attributes}} and {{SanitizerConfig/dataAttributes}} set</th>
+</tr>
+<tr>
+ <th>local {{SanitizerElementNamespaceWithAttributes/attributes}}</th>
+ <td class=long>All [=custom data attributes=] are allowed. No [=custom data attributes=] may
+   be listed in any allow-list, as that would mean a duplicate entry.</td>
+</tr>
+<tr>
+ <th>local {{SanitizerElementNamespaceWithAttributes/removeAttributes}}</th>
+ <td class=long>A [=custom data attribute=] is allowed, unless it's listed in the local
+   remove-list.
+   No [=custom data attribute=] may be listed in the global allow-list, as that would mean a
+   duplicate entry.</td>
+</tr>
+</table>
+
+Putting these rules in words:
+
+* Duplicates and interactions between global and local lists:
+  * If a global {{SanitizerConfig/attributes}} allow list exists, then all element's local lists:
+    * If a local {{SanitizerElementNamespaceWithAttributes/attributes}} allow lists exists,
+        there may be no duplicate entries between these lists.
+    * If a local {{SanitizerElementNamespaceWithAttributes/removeAttributes}} remove lists exists,
+        then all its entries must also be listed in the global {{SanitizerConfig/attributes}}
+        allow list.
```

Comment:
Could/should we mention somewhere here or somewhere else that this implies that there can be no duplicates between "local" `attributes` and `removeAttributes` lists?